script/install-as-user

```diff
@@ -105,6 +105,18 @@ if [ $USE_RBENV = true ]; then
   gem install bundler
 fi
 
+# Prevent extreme deprecation warnings on Ruby 2.7.0 provided by Ubuntu Focal
+# (Fixed in Ruby 2.7.2). ensure_line_present is not available here, so just do
+# this as it's pretty temporary.
+if ruby -v | grep "2.7.[01]" > /dev/null && ! grep "RUBYOPT" "$BASHRC" > /dev/null
+then
+  cat >>$BASHRC <<EOF
+if [ -x \$(which ruby) ] && ruby -v | grep "2.7.[01]" > /dev/null; then
+  export RUBYOPT='-W:no-deprecated -W:no-experimental'
+fi
+EOF
+fi
+
```

Comment:
Instead of grepping for "RUBYOPT" I would grep for some speical comment we include as this will allow us to include other RUBYOPT in the future if we ever needed to.

```suggestion
# Prevent extreme deprecation warnings on Ruby 2.7.0 provided by Ubuntu Focal
# (Fixed in Ruby 2.7.2). ensure_line_present is not available here, so just do
# this as it's pretty temporary.
RUBY27_OPT_COMMENT="Set Ruby options for Ruby 2.7"
RUBY27_OPT_START="# START $RUBY27_OPT_COMMENT"
RUBY27_OPT_END="# END $RUBY27_OPT_COMMENT"
if ruby -v | grep "2.7.[01]" > /dev/null && ! grep $RUBY27_OPT_START "$BASHRC" > /dev/null
then
  cat >>$BASHRC <<EOF
$RUBY27_OPT_START
if [ -x \$(which ruby) ] && ruby -v | grep "2.7.[01]" > /dev/null; then
  export RUBYOPT='-W:no-deprecated -W:no-experimental'
fi
$RUBY27_OPT_END
EOF
fi

```

Reply:
Good call. FYI, needed to quote the grep:

```diff
- if ruby -v | grep "2.7.[01]" > /dev/null && ! grep $RUBY27_OPT_START "$BASHRC" > /dev/null
+ if ruby -v | grep "2.7.[01]" > /dev/null && ! grep "$RUBY27_OPT_START" "$BASHRC" > /dev/null
```